cl/tests/cases.py

```diff
@@ -179,6 +180,21 @@ def tearDown(self) -> None:
         self.restart_celery_throttle_key()
         super().tearDown()
 
+    def assert_es_feed_content(self, node_tests, response, namespaces):
+        xml_tree = etree.fromstring(response.content)
+        for test_content, count in node_tests:
+            node_count = len(
+                xml_tree.xpath(test_content, namespaces=namespaces)
+            )  # type: ignore
+            self.assertEqual(
```

Comment:
This is a good time for subtests, yeah? https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Reply:
Yeah, I changed it to use `subTest`.